apollo-portal/src/main/java/com/ctrip/framework/apollo/portal/repository/PermissionRepository.java

```diff
@@ -21,4 +23,12 @@
    */
   List<Permission> findByPermissionTypeInAndTargetId(Collection<String> permissionTypes,
                                                      String targetId);
+
+  @Query("select p.id from Permission p where p.targetId = ?1 or p.targetId like CONCAT(?1, '+%')")
+  List<Integer> findPermissionId(String appId);
```

Comment:
we don't need a query like this since in the program, we do know the targetids, so we could call `findByPermissionTypeInAndTargetId` instead.